api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java

```diff
@@ -51,7 +51,7 @@ public static Meter getInstance() {
   public DoubleCounter.Builder doubleCounterBuilder(String name) {
     Utils.checkNotNull(name, "name");
     Utils.checkArgument(
-        StringUtils.isPrintableString(name) && name.length() <= NAME_MAX_LENGTH,
+        StringUtils.isValidMetricName(name) && name.length() <= NAME_MAX_LENGTH,
```

Comment:
Should we move the length of the name as well in the isValid method? Can be done in a separate PR.

Reply:
It makes sense to me